app/components/minister-position-function-select.js

```diff
@@ -0,0 +1,22 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+
+export default class MinisterPositionFunctionSelectComponent extends Component {
+  @service fastboot;
+  @service store;
+
+  constructor() {
+    super(...arguments);
+
+    if (!this.fastboot.isFastBoot) {
+      this.fetchMinisterPositionFunctionsTask.perform();
+    }
+  }
+
+  @task *fetchMinisterPositionFunctionsTask() {
+    return yield this.store.query('minister-position-function', {
+      'page[size]': 1000,
+    });
+  }
```

Comment:
As you suggested before, I don't think it will have more than 20 minister-position- functions.